src/lib/index.ts

```diff
@@ -1,7 +1,38 @@
 import fetch from 'node-fetch';
 import path from 'path';
 import execSh from 'exec-sh';
-import { specmaticJarPathLocal } from '../config';
+import { specmaticJarPathLocal, specmatic } from '../config';
+import fs from 'fs';
+import { check } from 'yargs';
+
+type Environment = Record<string, string>
+
+export const setSpecmaticEnvironement = (environmentName: string, environment: Environment) => {
+  console.log("Reading the file from -> ", path.resolve(specmatic))
+  let file = null
+
+  try {
+    file = require(path.resolve(specmatic))
+    for (var x in environment) { file.environments[environmentName].variables[x] = environment[x] }
+    fs.writeFileSync(path.resolve(specmatic), JSON.stringify(file, null, 2))
+    return checkSpecmaticEnvironment(environmentName, environment)
+  } catch (e) {
+    if (e instanceof TypeError) { console.log(e.toString()) }
```

Comment:
What is the purpose of this if-else? In any case, we are printing the error right? Is there some other kind of error you are expecting here?

Reply:
Mondule_Not_found, TypeError, but you are right this should be generic .There can be other kinds of error like the datatype of parameter passed is not proper or maybe something else